host_core/lib/host_core/application.ex

```diff
@@ -26,4 +32,67 @@ defmodule HostCore.Application do
     Logger.info("Starting Host Core")
     Supervisor.start_link(children, opts)
   end
+
+  defp retrieve_nats_env() do
+    rpc_host =
+      case System.get_env("WASMCLOUD_RPC_HOST") do
+        nil -> '0.0.0.0'
+        host -> host
+      end
+
+    rpc_port =
```

Comment:
You can supply default values to calls to `get_env` and avoid the case syntax entirely, see the [examples](https://hexdocs.pm/elixir/1.12/System.html#get_env/2-examples)